fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerSimpleTest.java

```diff
@@ -736,32 +736,6 @@ public void testOnlineIndexerBuilderWriteLimitBytes() throws Exception {
         final FDBStoreTimer timer = new FDBStoreTimer();
 
         try (FDBRecordContext context = openContext()) {
-            recordStore.checkVersion(null, FDBRecordStoreBase.StoreExistenceCheck.ERROR_IF_NOT_EXISTS).join();
-
-            // Build in this transaction.
-            try (OnlineIndexer indexer =
-                         OnlineIndexer.newBuilder()
-                                 .setRecordStore(recordStore)
-                                 .setTimer(timer)
-                                 .setIndex("newIndex")
-                                 .setLimit(100000)
-                                 .setMaxWriteLimitBytes(1)
-                                 .build()) {
-                // this call will "flatten" the staccato iterations to a whole range. Testing compatibility.
-                indexer.rebuildIndex(recordStore);
-            }
-            recordStore.markIndexReadable("newIndex").join();
-
-            assertEquals(numRecords, timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_SCANNED));
-            assertEquals(numRecords, timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RECORDS_INDEXED));
-
-            assertEquals(numRecords - 1, timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RANGES_BY_SIZE));
-            assertEquals(1, timer.getCount(FDBStoreTimer.Counts.ONLINE_INDEX_BUILDER_RANGES_BY_COUNT)); // last item
```

Comment:
These assertions should be kept, but on the block below, after rebuilding.

Reply:
✅